datalake/src/main/java/com/sequenceiq/datalake/service/startstop/DatabaseStartStopSupportService.java

```diff
@@ -0,0 +1,26 @@
+package com.sequenceiq.datalake.service.startstop;
+
+import com.sequenceiq.cloudbreak.common.mappable.CloudPlatform;
+import com.sequenceiq.datalake.entity.SdxCluster;
+import com.sequenceiq.datalake.service.EnvironmentClientService;
+import com.sequenceiq.environment.api.v1.environment.model.response.DetailedEnvironmentResponse;
+import org.apache.logging.log4j.util.Strings;
+import org.springframework.stereotype.Component;
+
+import javax.inject.Inject;
+
+@Component
+public class DatabaseStartStopSupportService {
+
+    @Inject
+    private EnvironmentClientService environmentClientService;
+
+    public boolean supportsDatabaseStartStop(SdxCluster sdxCluster) {
+        if (sdxCluster.hasExternalDatabase() && Strings.isNotEmpty(sdxCluster.getDatabaseCrn())) {
+            DetailedEnvironmentResponse environment = environmentClientService.getByCrn(sdxCluster.getEnvCrn());
+
+            return CloudPlatform.AWS.name().equalsIgnoreCase(environment.getCloudPlatform());
```

Comment:
can this be part of the resourceconnector ?  I think that is much flexible so create a method there like databasePauseSupported()

Reply:
As we have discussed, it can't be a part of the ResourceConnector but it got moved to the PlatformConfig.